test/backend/datastores.mock.spec.js

```diff
@@ -15,19 +15,21 @@ import {
 
 
 let servers;
-let connectionId;
+let connectionId = 'postgres-189ebfb4-e1b4-446c-9b83-b326875fa2d8';
```

Comment:
`const`?

Reply:
is there an easy way to check that all the connections in `mock-storage` have been read?

Reply:
Updated.. !!